modules/manually-creating-alibaba-ram-user.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_alibaba/manually-creating-alibaba-ram.adoc
+
+:_content-type: PROCEDURE
+[id="manually-creating-alibaba-ram-user_{context}"]
+= Creating the required RAM user
+
+// https://github.com/openshift/cloud-credential-operator/pull/412/files#diff-2480a11ca4927139d6eaa9883946b6f4cb38358cd98def8c57dd73e9319dbc9cR232
+
+When you configure the Resource Access Management (RAM) user, be sure to consider the following requirements:
+
+* Attach the `AdministratorAccess` and `AliyunBSSFullAccess` policies to a RAM user to ensure that the account has sufficient permission to create the {product-title} cluster. This policy grants permissions to manage all Alibaba Cloud resources. The user must have the following permissions:
++
+** ram:CreatePolicy
+** ram:GetPolicy
+** ram:CreatePolicyVersion
+** ram:DeletePolicy
+** ram:DetachPolicyFromUser
+** ram:ListPoliciesForUser
+** ram:AttachPolicyToUser
+** ram:CreateUser
+** ram:GetUser
+** ram:DeleteUser
+** ram:CreateAccessKey
+** ram:ListAccessKeys
+** ram:DeleteAccessKey
+** bss:CreateInstance
+** bss:Query
+** bss:ModifyInstance
+
+* Record the RAM AccessKey (AK) for this user. For more information, see link:https://www.alibabacloud.com/help/en/doc-detail/53045.htm[Obtain an AccessKey pair].
+
+* Add the AcessKey ID (`access_key_id`) and AccessKey secret (`access_key_secret`) into the link:https://github.com/aliyun/alibaba-cloud-sdk-go/blob/master/docs/2-Client-EN.md#2-credentials-file[`~/.alibabacloud/credentials` file] on your local computer.  The Cloud Credential Operator (CCO) utility uses these credentials when processing `Credential Request` objects.
```

Comment:
_The Cloud Credential Operator (CCO) utility uses these credentials when processing `Credential Request` objects._

@kwoodson @mtulio @joelddiaz Is this correct? This seemed to be the behavior when I attempted to use the `create-ram-users` command.

Reply:
If so, do we have a more realistic credentials file?

Reply:
Yes. The `ccoctl` tool will use the credentials stored in that file to do what it needs to do. Not sure what a 'more realistic credentials file' is... The format looks like:
```
[default]
type = access_key
access_key_id = AccessKeyHere
access_key_secret = AccessKeySecretHere
```

Reply:
In addition to `Cloud Credential Operator (CCO) utility` I believe we should also mention `ccoctl`.